### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by marimeireles**
Initial docs for memestra.
This PR contains user docs only.

AFAIC it's still missing the explanations for `--recursive`.
Maybe something else too?

Thanks @serge-sans-paille !

### CODE REVIEW

**Comment by marimeireles** [comment]
Oh no, I pushed the build too...
One sec I'll undo it.

**Comment by serge-sans-paille** [comment]
Please remove the `_build` dir ;-)

File: memestra/version.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+__version__ = '0.0.1'
```
Comment (serge-sans-paille) [inline memestra/version.py:1]: Nice!

File: docs/installation.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
+.. Copyright (c) 2020, Serge Guelton, Johan Mabille, and Mariana Meireles
+
+   Distributed under the terms of the BSD 3-Clause License.
+
+   The full license is in the file LICENSE, distributed with this software.
+
+Installation
+============
+
+From Pypi
```
Comment (serge-sans-paille) [inline docs/installation.rst:10]: PyPI

File: memestra/version.py (hunk 1 above)
Comment (marimeireles) [reply memestra/version.py:1]: Actually is 0.0.3 now right :p
I'll fix it!

**Comment by marimeireles** [comment]
Getting this error in the build:
`Solving environment: ...working... failed with repodata from current_repodata.json, will retry with next repodata source.`
`The command "conda create -q -n test-environment -c conda-forge python=$TRAVIS_PYTHON_VERSION pandoc" failed and exited with 1 during .` Just a Travis hiccup maybe?

File: docs/memestra-cache.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
+Memestra-cache
+==============
+
+To handle the cache, memestra installs a tool named ``memestra-cache``.
```
Comment (serge-sans-paille) [inline docs/memestra-cache.rst:4]: Memestra uses a cache to speedup some computations. To handle it, it installs a tool named...

File: docs/memestra-cache.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Memestra's caching infrastructure is a file-based cache, located in ~/.memestra (RW) and in memestra/cache (RO). The key is a hash of the file content and the value is deprecation information, generator used, etc.
```
Comment (serge-sans-paille) [inline docs/memestra-cache.rst:6]: `~/.memestra` (formating). Same for `memestra/cache` which should probably `<prefix>/memestra/cache`

File: docs/memestra-cache.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+To interact with memestra cache:
+
+**Positional arguments:**
+
+``-set``
+
+Set a cache entry
+
+`-list``
```
Comment (serge-sans-paille) [inline docs/memestra-cache.rst:18]: nit: Missing back-tick

**Pull request merged**